traffic_ops/testing/api/v4/servercapabilities_test.go

```diff
@@ -83,6 +96,29 @@ func GetTestServerCapabilities(t *testing.T) {
 	}
 }
 
+func UpdateTestServerCapabilitiesWithHeaders(t *testing.T, header http.Header) {
+	resp, _, err := TOSession.GetServerCapabilitiesWithHdr(header)
+	if err != nil {
+		t.Fatalf("Expected no error, but got %v", err.Error())
+	}
+	if len(resp) == 0 {
+		t.Fatal("no server capability in response, quitting")
+	}
+	originalName := resp[0].Name
+	newSCName := "sc-test"
+	resp[0].Name = newSCName
+
+	if (resp) != nil {
+		_, reqInf, err := TOSession.UpdateServerCapabilityByNameWithHdr(originalName, &resp[0], header)
```

Comment:
Also, I noticed that in some functions, the header is passed as the second argument, and in some, it is passed in as the last argument. It would be nice to keep it consistent.

Reply:
Header is passed as the last argument.. a function may have one, two or three (etc) variables being passed, and header is the last variable. 
UpdateTopologyWithHdr(name string, t tc.Topology, header http.Header)
UpdateServerCapabilityByNameWithHdr(name string, sc *tc.ServerCapability, header http.Header)
GetTopologyWithHdr(name string, header http.Header)
GetServerCapabilityWithHdr(name string, header http.Header)
GetServerCapabilitiesWithHdr(header http.Header)